### PR TITLE
Populate the environment with AWS credentials

### PR DESCRIPTION
**Pull request by maurodoglio**
We need to do this because Boto doesn't know about our .env file.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/32)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: atmo/settings.py [hunk 1]
```diff
@@ -108,6 +108,10 @@
     'PUBLIC_DATA_BUCKET':  'telemetry-public-analysis-2',
     'PRIVATE_DATA_BUCKET': 'telemetry-private-analysis-2',
 }
+for aws_cred in ('AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'):
```
Comment (jezdez) [inline atmo/settings.py]: Let's add `AWS_DEFAULT_REGION` as well

Comment (maurodoglio) [inline atmo/settings.py]: 👍

File: atmo/settings.py [hunk 2]
```diff
@@ -108,6 +108,10 @@
...
+    if not aws_cred in os.environ:
```
Comment (maurodoglio) [inline atmo/settings.py]: 👍

**Pull request merged**